conf/evolutions/reversions/080-job-add-cancelled.sql

```diff
@@ -0,0 +1 @@
+-- unfortunately, ALTER TYPE ... ADD cannot not be reverted
```

Comment:
not sure what the original idea was? there is simply no point of return after that migration? only from backups?

Reply:
Well, the old scala code doesn’t care if the SQL enum has another possible value, as long as it doesn’t actually have to parse any rows with it, so we could have rolled back that commit in case of a bug, without changing the sql schema back.
The reversions are mostly there as a safeguard in case of bugs with the new code that show up immediately after deploying. So it’s not always worth going to the trouble of building a 100% accurate reversion, rather just enough to let the old code run again until we can repair the new.
That is also why the correctness of the reversions themselves is not asserted in the CI.

Reply:
Ok :+1:

> That is also why the correctness of the reversions themselves is not asserted in the CI.

does this mean you are against the CI check I added? it doesn't assert correctness but rather "completeness". like in the above case, one can always create an empty file with a comment for such special cases. in my opinion this would be a good practice, as it forces us to at least think about the reversion (and one cannot forget about it). but i have no strong opinion here. I can remove it again.

Reply:
Thanks for the comment! Fixed in baa129c